htlcswitch/switch.go

```diff
@@ -345,6 +363,29 @@ type resolutionMsg struct {
 	doneChan chan struct{}
 }
 
+// RequestAlias is called by outside subsystems so that they can request a
+// channel alias. By managing the aliases, we can ensure that no two aliases
+// are the same. An error is returned if the request failed.
+func (s *Switch) RequestAlias() (lnwire.ShortChannelID, error) {
```

Comment:
I would think alias management/generation would be done at either the funding manager or invoice registry level?

Reply:
I didn't know where to put this